eq-author-api/middleware/auth.js

```diff
@@ -2,14 +2,14 @@ const { isNil, isEmpty } = require("lodash/fp");
 const jwt = require("jsonwebtoken");
 
 module.exports = (logger, context) => (req, res, next) => {
-  const authHeader = req.header("authorization");
+  const authHeader = req.header(process.env.AUTH_HEADER_KEY || "authorization");
   if (isNil(authHeader)) {
     logger.error("Request must contain a valid authorization header.");
     res.send(401);
     return;
   }
 
-  const accessToken = authHeader.replace("Bearer ", "");
+  const accessToken = authHeader.replace("Bearer ", "").replace(/=/g, "");
```

Comment:
Why is this necessary? Can we get a test around it as well.

Reply:
Looks like conflicting interpretations of the jwt spec

Reply:
This seems wrong - its completely acceptable for a base64url encoded value to contain `=`. This needs more investigation.

Reply:
Yeh that seems to be the issue. each of the parts of a JWT is base64 encoded but some of the JWT libraries don't allow the `=`

Reply:
https://github.com/auth0/node-jsonwebtoken/issues/162

Reply:
Can you add that into the code and add a test to cover this case. Just so its obvious. I don't mind if the link to the issue is only on the test.